mlflow/store/file_store.py

```diff
@@ -31,18 +31,39 @@ def _default_root_dir():
     return get_env(_TRACKING_DIR_ENV_VAR) or os.path.abspath(DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH)
 
 
+def _make_persisted_experiment_dict(experiment):
+    exp_dict = dict(experiment)
+    try:  # Cast to int for backwards compatibility with type change for run submit
+        exp_dict["experiment_id"] = int(exp_dict["experiment_id"])
+    except ValueError:
+        pass
+    return exp_dict
+
+
+def _read_persisted_experiment_dict(experiment_dict):
+    dict_copy = experiment_dict.copy()
```

Comment:
is copy() enough or do we want to consider deep_copy()?

Reply:
For now it should be enough since the Experiment is flat. Although I agree that exposing this in the case Experiment is no longer flat would be very problematic